src/extensions/default/bramble/lib/launcher.js

```diff
@@ -28,7 +28,14 @@ define(function (require, exports, module) {
         var server = this.server;
         var browser = this.browser;
 
-        browser.update(server.serveLiveDoc(url));
+        server.serveLiveDoc(url, function(err, urlOrHTML) {
+            if(err) {
+                // TODO: how to deal with this error?                
+                console.error("[Launcher Error]", err);
```

Comment:
I don't love this, is there a better thing we can do here?  Should we serve a 500 or something?